tensorflow/contrib/tensorrt/test/quantization_mnist_test.py

```diff
@@ -0,0 +1,197 @@
+      logits_out = tf.import_graph_def(graph_def,
+          input_map={INPUT_NODE_NAME: features},
+          return_elements=[OUTPUT_NODE_NAME+':0'],
+          name='')[0]
+    loss = tf.losses.sparse_softmax_cross_entropy(
+        labels=labels,
+        logits=logits_out)
+    tf.summary.scalar('loss', loss)
+    classes_out = tf.argmax(logits_out, axis=1, name='classes_out')
+    accuracy = tf.metrics.accuracy(
+        labels=labels,
+        predictions=classes_out,
+        name='acc_op')
+    tf.summary.scalar('accuracy', accuracy[1])
+    if mode == tf.estimator.ModeKeys.EVAL:
+      return tf.estimator.EstimatorSpec(
+          mode,
+          loss=loss,
+          eval_metric_ops={'accuracy': accuracy})
+    elif mode == tf.estimator.ModeKeys.TRAIN:
+      optimizer = tf.train.AdamOptimizer(learning_rate=1e-2)
+      train_op = optimizer.minimize(
+          loss,
+          global_step=tf.train.get_global_step())
+      return tf.estimator.EstimatorSpec(
+          mode,
+          loss=loss,
+          train_op=train_op)
+
+  tf_config = tf.ConfigProto()
+  tf_config.gpu_options.allow_growth = True
+  estimator = tf.estimator.Estimator(
+      model_fn=model_fn,
+      model_dir=model_dir,
+      config=tf.estimator.RunConfig(session_config=tf_config))
+  if is_training:
+    estimator.train(train_input_fn)
+  results = estimator.evaluate(eval_input_fn)
+  print('accuracy:', results['accuracy'])
+  return results
+
+def get_graph_def(use_trt, batch_size, model_dir):
+  # Load graph and freeze
+  with tf.Graph().as_default() as graph:
+    with tf.Session() as sess:
+      x = tf.placeholder(shape=(None, 28, 28, 1),
+                         dtype=tf.float32,
+                         name=INPUT_NODE_NAME)
+      logits_out = build_graph(x)
+      # Load weights
+      saver = tf.train.Saver()
+      checkpoint_file = tf.train.latest_checkpoint(model_dir)
+      saver.restore(sess, checkpoint_file)
+      # Freeze
+      graph_def = tf.graph_util.convert_variables_to_constants(
+          sess,
+          sess.graph_def,
+          output_node_names=[OUTPUT_NODE_NAME]
+      )
+  # Convert with TF-TRT
+  if use_trt:
+    print('nodes before:', len(graph_def.node))
+    graph_def = trt.create_inference_graph(graph_def,
+        outputs=[OUTPUT_NODE_NAME],
+        max_batch_size=batch_size,
+        precision_mode='int8',
+        max_workspace_size_bytes=4096 << 19,
+        minimum_segment_size=2,
+        use_calibration=False,
+    )
+    print('tftrt total nodes:', len(graph_def.node))
+    print('trt only nodes',
+        len([1 for n in graph_def.node if str(n.op)=='TRTEngineOp']))
+  return graph_def
+
+
+class QuantizationAwareTrainingMNISTTest(test_util.TensorFlowTestCase):
+
+  def testEval(self):
+    acc_tf = run(is_training=False,
+        use_trt=False,
+        batch_size=128,
+        num_epochs=None,
+        model_dir='./quantization_mnist_test_data')['accuracy']
```

Comment:
Please add a build rule for the test data and access it maybe using `test.test_src_dir_path()`. There is an example [BUILD](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/python/kernel_tests/BUILD#L3021) and [py](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/python/kernel_tests/self_adjoint_eig_op_test.py#L78) file you can follow.